gcloud-java-contrib/gcloud-java-nio/pom.xml

```diff
@@ -77,24 +72,6 @@
       <artifactId>mockito-core</artifactId>
       <version>1.9.5</version>
     </dependency>
-    <dependency>
```

Comment:
do you think we need to create another pom (maybe another module under contrib) that would function as an aggregate jar (like the gcloud-java module) for all contribs? If not and we think that each contrib should be
a separate dependency only then we need to change the README.md in gcloud-java-contrib